### PR TITLE
Feature/URP-2DLights-PostProcessing-NewspaperUI Hook

### PR DESCRIPTION
**Pull request by SabrinaTheCookie**
**NOTE: THIS BRANCH HAS UPGRADED TO URP. TEST BEFORE MERGING**
- URP
- 2D Lights
- Title Screen Spruce up
- Global Lighting
- Post Processing URP-ified.
- Added newspaperUI hook to reviews.

### CODE REVIEW

File: Assets/_Scripts/Gameplay/ReviewDataSO.cs [hunk 1]
```diff
@@ -10,6 +10,15 @@ public class ReviewDataSO : ScriptableObject
...
         public AffinityFeedback affinityFeedback;
+        public ReviewImageChoices reviewImageChoices;
+        public int stars;
+        public string date;
+        public string caption;
+        public int issueNumber;
+        public int volNumber;
+        public string reviewTitle;
+        public string reviewSubTitle;
+        public Sprite reviewImage;
```
Comment (JamesCoburn246) [inline Assets/_Scripts/Gameplay/ReviewDataSO.cs:21]: Did these need to be added here? They will be fixed per-night if so.

File: Assets/_Scripts/Managers/ReviewManager.cs [hunk 2]
```diff
@@ -40,6 +42,10 @@ public StarRating GetPerformanceRating()
         {
             CalculateRating();
         }
+
+        //TODO I need the real ReviewDataSO.
```
Comment (JamesCoburn246) [inline Assets/_Scripts/Managers/ReviewManager.cs:46]: One for you to TODO maybe @baconeta?

File: Assets/_Scripts/Gameplay/ReviewDataSO.cs [hunk 3]
```diff
@@ -10,6 +10,15 @@ public class ReviewDataSO : ScriptableObject
         public MusicianAppropriatenessFeedback musicianChoiceFeedback;
         public InstrumentFeedback instrumentChoiceFeedback;
         public AffinityFeedback affinityFeedback;
+        public ReviewImageChoices reviewImageChoices;
+        public int stars;
```
Comment (JamesCoburn246) [inline Assets/_Scripts/Gameplay/ReviewDataSO.cs:14]: Note that stars are originally an Enum. Maybe this should be renamed to be "half-stars" to be more descriptive.

File: Assets/_Scripts/UI/NewspaperUI.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        image.sprite = review.reviewImage;
+        //TODO Set stars based on the review score
+        //0 = 0 star
+        //1 = .5 star
+        //2 = 1 star
+        //10 = 5 star
+        for(int i = 0; i < review.stars; i++)
+        {
+            //Set sprite to stars score
+            //stars[i].sprite = starSprites?;
```
[the remark this answers is not in the document]
Comment (SabrinaTheCookie) [reply Assets/_Scripts/UI/NewspaperUI.cs:44]: We do have no/half/full star assets

File: Assets/_Scripts/Gameplay/ReviewDataSO.cs (hunk 3 above)
Comment (SabrinaTheCookie) [reply Assets/_Scripts/Gameplay/ReviewDataSO.cs:14]: Changed to half-stars

File: Assets/_Scripts/Gameplay/ReviewDataSO.cs (hunk 1 above)
Comment (SabrinaTheCookie) [reply Assets/_Scripts/Gameplay/ReviewDataSO.cs:21]: Idk if review title == performance title. Ill let @baconeta  decide.

**Comment by SabrinaTheCookie** [comment]
@JamesCoburn246 @baconeta  This is ready

**Review by baconeta — APPROVED** [review]
I will look at TODOs in an hour or so

**Pull request merged**